architecture/integration-service.md

```diff
@@ -5,21 +5,20 @@
 
 The Integration Service is composed of controllers that facilitate automated testing of content produced by the build pipelines. It is mainly responsible for integration testing capabilities.
 
-The Integration service uses the pipeline, snapshot, and environment controllers to watch for Component builds, trigger and manage testing Tekton Pipelines, and create releases based on the testing outcome(s).
+The Integration service uses the pipeline, snapshot, and component controllers to watch for Component builds, trigger and manage testing Tekton Pipelines, and create releases based on the testing outcome(s).
 
 ### Goals
 
-- The Integration service need to be able to deploy a specific set of images to an environment and record the results of integration testing.
-- Given a specific set of images, Integration service should be able to tell if they passed application validation tests.
 - The Integration service should be able to update one component image at a time in order to test and deploy individual component builds.
+- The Integration service need to be able to test the snapshot created for component image and record the results of integration testing.
+- Given a specific set of images, Integration service should be able to tell if they passed application validation tests.
 - The Integration service should have a capability to automatically promote sets of passed images.
 - To be SLSA compliant, Integration service should also be able to tell what version of the build and test pipelines were used to create each component image.
 
 ### High Level Workflow
 
 - When a build pipeline completes, Integration service creates a Snapshot CR representing a new collection of components that should be tested together.
-- When Integration Service sees a new Snapshot CR (created either by itself or by a user), it coordinates deployment of the application for testing. It does this by creating new “ephemeral” Konflux Environment CRs which trigger the GitOps Service to provision the ephemeral test environment.
-- After the environment is ready and the application snapshot has been deployed to it, Integration Service tests and validates the application according to user-provided configuration. It does this by executing Tekton PipelineRuns against the ephemeral test Environment.
+- When Integration Service sees a new Snapshot CR (created either by itself or by a user),  it tests and validates the application according to user-provided configuration. It does this by executing Tekton PipelineRuns.
```

Comment:
I think there are two whitespaces after comma.
Btw this line describes integration-tests?

Reply:
Yes, remove them since https://github.com/konflux-ci/architecture/pull/182 has included them.